extendedcopy.go

```diff
@@ -182,11 +179,31 @@ func findRoots(ctx context.Context, storage content.ReadOnlyGraphStorage, node o
 // For performance consideration, when using both FilterArtifactType and
 // FilterAnnotation, it's recommended to call FilterArtifactType first.
 func (opts *ExtendedCopyGraphOptions) FilterAnnotation(key string, regex *regexp.Regexp) {
+	filter := func(predecessors []ocispec.Descriptor, filtered []ocispec.Descriptor) []ocispec.Descriptor {
```

Comment:
It is not nil since the second page of the result of `Referrers`. See L200.
https://github.com/oci-playground/oras-go/blob/c14a47153be47e4713e2f6a6d6b15d428e30cc97/extendedcopy.go#L199-L200.